index.html

```diff
@@ -0,0 +1,626 @@
+<!DOCTYPE html>
```

Comment:
- [ ] Kindly fix the HTML file so you don't have any warnings.

> Warnings
![image](https://user-images.githubusercontent.com/57414789/82961712-c9006580-9f83-11ea-8889-dee4fbdd8ca2.png)

Reply:
:100: